jaq-native/src/lib.rs

```diff
@@ -260,6 +269,69 @@ fn now() -> Result<f64, Error> {
 #[cfg(feature = "std")]
 const STD: &[(&str, usize, RunPtr)] = &[("now", 0, |_, _| box_once(now().map(Val::Float)))];
 
+#[cfg(feature = "math")]
+const MATH: &[(&str, usize, RunPtr)] = &[
+    math::f_f!(acos),
+    math::f_f!(acosh),
+    math::f_f!(asin),
+    math::f_f!(asinh),
+    math::f_f!(atan),
+    math::f_f!(atanh),
+    math::f_f!(cbrt),
+    math::f_f!(cos),
+    math::f_f!(cosh),
+    math::f_f!(erf),
+    math::f_f!(erfc),
+    math::f_f!(exp),
+    math::f_f!(exp10),
+    math::f_f!(exp2),
+    math::f_f!(expm1),
+    math::f_f!(fabs),
+    math::f_fi!(frexp),
+    math::f_f!("gamma", tgamma),
```

Comment:
How do you know that gamma corresponds to tgamma?

Reply:
This seems to be another pitfall I fell into. Consider the [man page](https://man7.org/linux/man-pages/man3/gamma.3.html):

> Because of historical variations in behavior across systems, this function is not specified in any recent standard. 4.2BSD had a gamma() that computed ln(|Gamma(|x|)|), leaving the sign of Gamma(|x|) in the external integer signgam.  In 4.3BSD the name was changed to [lgamma(3)](https://man7.org/linux/man-pages/man3/lgamma.3.html), and the man page promises
>
> "At some time in the future the name gamma will be rehabilitated and used for the Gamma function"
>
> This did indeed happen in 4.4BSD, where gamma() computes the Gamma function (with no effect on signgam).  However, this came too late, and we now have [tgamma(3)](https://man7.org/linux/man-pages/man3/tgamma.3.html), the "true gamma" function.

I tested this with the statically linked jq that can be downloaded from <https://jqlang.github.io/jq/>, and compared it to gojq. This is the result:

```text
$ jq -nc '1.5, 2.5, 3.5 | [tgamma, gamma, lgamma]'
[0.886226925452758,-0.12078223763524522,-0.12078223763524522]
[1.329340388179137,0.2846828704729192,0.2846828704729192]
[3.3233509704478426,1.2009736023470743,1.2009736023470743]
```

Vs.

```text
$ gojq -nc '1.5, 2.5, 3.5 | [tgamma, gamma, lgamma]'
[0.8862269254527579,0.8862269254527579,-0.12078223763524522]
[1.329340388179137,1.329340388179137,0.2846828704729192]
[3.323350970447843,3.323350970447843,1.2009736023470743]
```

So depending on how backwards-compatible we want to be, we should either make `gamma` a synonym of `lgamma` or `tgamma` 🥴

Reply:
In any case, we do not have to declare gamma here because we could just declare it in jaq-std. However, I propose for now that we simply do not declare gamma at all, until it is clarified what it should evaluate to.